src/manager.cpp

```diff
@@ -120,6 +121,35 @@ sdbusplus::async::task<> Manager::startSyncEvents()
     co_return;
 }
 
+// NOLINTNEXTLINE
+void Manager::syncData(const config::DataSyncConfig& dataSyncCfg)
+{
+    using namespace std::string_literals;
+    //  TODO:
```

Comment:
Line 128 and 129 is not required since you added a TODO in the required place (line number: 138)